src/python/pants/engine/process.py

```diff
@@ -30,6 +30,9 @@
 logger = logging.getLogger(__name__)
 
 
+BASH_SEARCH_PATHS = ("/usr/bin", "/bin", "/usr/local/bin")
```

Comment:
Could you please update `archive.py` to use this value, rather than duplicating? Will keep the surface area smaller.

Maybe rename to something more generic, although I can't think of a good name for the binaries that we expect to be installed in `/bin` et al.

Reply:
The `archive.py` search path arguably should be configurable (as commented there), since it's used for multiple tools. This one on the other hand does not seem like it should be, necessarily, as we're using it to bootstrap.